react/Viewer/index.jsx

```diff
@@ -107,6 +108,8 @@ export class Viewer extends Component {
             file={currentFile}
             onClose={this.onClose}
             renderFallbackExtraContent={renderFallbackExtraContent}
+            isOnlyOfficeEnabled={onlyOfficeProps && onlyOfficeProps.isEnabled}
+            onlyOfficeOpener={onlyOfficeProps && onlyOfficeProps.opener}
```

Comment:
We should not rename props on the fly. It's hard to follow and create conversion rules to remember. 

Why not having the same object than previously? aka 
onlyOfficeProps={{
isEnabled
...
}} 

?